test/suite/test_verbose01.py

```diff
@@ -58,13 +52,16 @@ def expect_verbose(self, categories, patterns):
         # operations on the connection (for generating the expected verbose output).
         yield conn
         # Read the contents of stdout to extract our verbose messages.
-        output = self.readStdout(self.nlines)
+        nlines = 30000
+        output = self.readStdout(nlines)
         # Split the output into their individual messages. We want validate the contents of each message
         # to ensure we've only generated verbose messages for the expected categories.
         verbose_messages = output.splitlines()
 
-        # If no categories were passed, we aren't expecting any output.
-        if len(categories) == 0:
+        # Check if we are expecting any output.
```

Comment:
Nitpick: This comment doesn't add much context and can be dropped

Reply:
Removed in ff5e87280fc27e87475afb6a82d0c949838c0ea6